jobbergate-cli/jobbergate_cli/render.py

```diff
@@ -116,13 +116,17 @@ def render_list_results(
     :param: hidden_fields: Columns that should (if not using ``full`` mode) be hidden in the ``Table`` output
     :param: title:         The title header to include above the ``Table`` output
     """
-    if envelope.pagination.total == 0:
-        terminal_message("There are no results to display", subject="Nothing here...")
-        return
-
     if ctx.raw_output:
-        render_json(envelope.results)
+        # THIS IS SO DUMB....need to dump to string to make sure paths are converted,
+        # then convert back to dict so we can strip out the "results" element, then
+        # dump back to string
+        serialized = envelope.json()
+        deserialized = json.loads(serialized)
+        render_json(deserialized["results"])
```

Comment:
why not to use regex validation and manipulation?

Reply:
JSON is a context-free grammar. It's a bad idea to parse it with regular expressions in general.
https://cstheory.stackexchange.com/a/3988

There's about a million PRs, issues, and discussions on pydantic about this issue. I'm still holding out hope that eventually they will add something like a `json_dict()` method on pydantic models for this very reason.